mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/MesosSchedulerClient.java

```diff
@@ -71,7 +75,7 @@ public MesosSchedulerClient(
     }
 
     @NotNull
-    public Observable<Protos.Event> openEventStream(@NotNull final Call subscription) {
+    public Observable<In> openEventStream(@NotNull final Out subscription) {
```

Comment:
Lets use `Send` and `Receive`. `Send` when referring to `Proto.Call` and `Receive` when referring to `Proto.Event`.